charts/common/templates/clusterrole.yaml

```diff
@@ -23,6 +23,7 @@ rules:
       - batch
       - discovery.k8s.io
       - metrics.k8s.io
+      - storage.k8s.io
```

Comment:
I suggest adding this to release notes that this version expands clusterrole permissions

Reply:
Done